app/src/components/main/grid/grid.jsx

```diff
@@ -33,13 +42,18 @@ Grid.propTypes = {
   sortingColumn: PropTypes.string,
   onChangeSorting: PropTypes.func,
   onFilterClick: PropTypes.func,
+  onDeleteItem: PropTypes.func,
+  onMoveToDebug: PropTypes.func,
+  onEditItem: PropTypes.func,
 };
 Grid.defaultProps = {
   columns: [],
   data: [],
   sortingDirection: null,
   sortingColumn: null,
-  onChangeSorting: () => {
-  },
+  onChangeSorting: () => {},
   onFilterClick: () => {},
+  onDeleteItem: () => {},
+  onMoveToDebug: () => {},
```

Comment:
Grid is a universal component. It shouldn't know anything about items it displays. 'Move to debug' operation is applicable only to launches, so it's not universal at all.